docs/user/integration/dynatrace/README.md

```diff
@@ -259,3 +259,29 @@ For easier access from the Kyma dashboard, add links to new navigation under **D
     ```
 
 2. If your Secret has a different name or namespace, then download the file first and adjust the namespace and name accordingly in the 'dataSources' section of the file.
+
+## Use Dynatrace Dashboards
+
+You can view logs, traces, and metrics in Dynatrace dashboards:
+
+- To view the status of the Dynatrace integration with the Kyma Telemetry module, go to Dashboards > Import Dashboard, and manually import the file [Telemetry Module Status](./telemetry-resource-metrics.json).
+    > [!WARNING]
+    > Kyma Telemetry resources contain the following custom OpenTelemetry resource attributes:
+    > - `k8s.resource.name`
+    > - `k8s.resource.group`
+    > - `k8s.resource.kind`
+    > - `k8s.resource.version`
+    > 
+    > For the dashboard to function properly, these attributes must be added to the allow list of OpenTelemetry metrics resource attributes. To make these configuration changes, go to Settings > Metrics > OpenTelemetry metrics > Allow list: resource and scope attributes.
+
+## Use Dynatrace Alerts
+
+You can create Dynatrace Alerts based on certain metric events to send alerts to your system of choice. 
+
+1. [Create a problem alerting profile](https://docs.dynatrace.com/docs/analyze-explore-automate/notifications-and-alerting/alerting-profiles#create-an-alerting-profile).
+2. Set up alert integration with your system of choice. For more information on how to set up alert integration with your system, go to the official [Dynatrace documentation](https://docs.dynatrace.com/docs/analyze-explore-automate/notifications-and-alerting/problem-notifications).
+3. Create a [metric event](https://docs.dynatrace.com/docs/discover-dynatrace/platform/davis-ai/anomaly-detection/set-up-a-customized-anomaly-detector/how-to-set-up/metric-events) by going to Settings > Anomaly Detection > Metric events > Add metric event. Define your metric event with a metric selector or a metric key.
+4. Add a custom event filter to your alerting profile in your problem alerting profile tab by clicking on the **Add event filter** button. Here you can select a custom filter based on the Metric event title, description, or property.
```

Comment:
Again, I hope Dynatrace is documenting how to do this and we can link to their doc.

```suggestion
4. Add a custom event filter to your alerting profile. You can select a custom filter based on the Metric event title, description, or property.
```